logbook-servlet/src/main/java/org/zalando/logbook/servlet/LocalResponse.java

```diff
@@ -108,6 +120,82 @@ private Tee tee() throws IOException {
         return tee;
     }
 
+    @Override
+    public int getStatus() {
+        return status;
+    }
+
+    @Override
+    public void setStatus(final int status) {
+        this.status = status;
+        super.setStatus(status);
+    }
+
+    @Override
+    public void sendError(final int status) throws IOException {
+        this.status = status;
+        super.sendError(status);
+    }
+
+    @Override
+    public void sendError(final int status, final String msg) throws IOException {
+        this.status = status;
+        super.sendError(status, msg);
+    }
+
+    @Override
+    public void addDateHeader(final String name, final long date) {
+        super.addDateHeader(name, date);
+        addMultiMapHeader(name, format(date));
```

Comment:
Format could be simplified with `Instant.ofEpochMilli()` instead of `new Date()`